### DISCUSSION TITLE
Successful installation on Orange Pi Zero2

### DISCUSSION THREAD

**Discussion by ameesme** [Nachtzuster/BirdNET-Pi#11] [Show and tell]
Hi @Nachtzuster,

Thank you for your effort in supporting BirdNET Pi. It's much appreciated.

This is just a ping to let other Orange Pi owners know that I was able to set up your fork on a Orange Pi Zero2 running the latest Debian image provided by Orange Pi (currently `Orangepizero2_3.1.0_debian_bookworm_server_linux6.1.31`). The only modification required was setting up passwordless sudo.

- Mees

**Comment by Nachtzuster** [comment]
@ameesme Thanks for the feedback. 
That board does look like a nice, efficient way to run BirdNET-Pi.
Even more broadly, any board running a 64bit Debian Bookworm based os, will most likely do these days. No guaranties though :wink: 
(I'm running [this](https://www.armbian.com/lepotato/) one)

**Reply by BAM-Team** [reply]
Hi all, here is my "quick an dirty" installation instruction for the Orange Pi Zero 2W. Maybe somebody will find it helpful...
Cheers, Carsten
[Github Birdnet OrangePI02w Installation EN.pdf](https://github.com/user-attachments/files/18177924/Github.Birdnet.OrangePI02w.Installation.EN.pdf)

I am not a linux pro - additions and improvements are welcome!

**Comment by lloydbayley** [comment]
Well that's encouraging! First (reported) install on a semi-alien system and it went smoothly! I think there's something in the fact that everything is now updated for the "current" releases of all packages. Everything just seems to get up and boogie!

**Comment by gutocarv** [comment]
Let me use this topic to say a huge thanks to @Nachtzuster for keep running the Rasp-Pi project.
After many tries,  with many installation fails in the original BirdNET-PI I've moved to this fork and everyting is running just fine

thank you very much, 
I have no much tech skills - but will try to help here as much as I can!!

**Comment by BAM-Team** [comment]
Hi @Nachtzuster...Than you for keeping the BirdNet dream alive :-D
I am just trying out the OrangePI Zero 2w and got it done with Patrick´s repo.
I will try an Installation with your repo, too
The OPI ist much better performing than RPI02w, so this will be a game changer for our monitoring projects
Thanks so far, I will keep you updated.
Cheers, Carsten

P.S.: Migrated to your Repo on a OPI02W with no problems! Excellent work!

**Comment by BAM-Team** [comment]
Hi @ameesme 
Great, thanks for the Info :-) I will try this out. 
cheers, Carsten

**Comment by MichaelPWalter** [comment]
@BAM-Team How does the wattage of the OrangePi compare to the RPi02W? Also where do you think the performance increase comes from?

**Comment by BAM-Team** [comment]
@MichaelPWalter 
Hi Michael, I don't have an exact comparison in terms of wattage. With a 20000mAh power bank I can run the OrangePi for about 40 hours. Performance: My OrangePi has 4GB memory, RPi02W 0.5 GB (!) - OrangePi CPU clock speed is 1.5 GHz instead of 1 GHz. The big performance advantage comes imho from the higher memory configuration. Cheers, Carsten

**Comment by UnviableFriend** [comment]
I have it running on an [Orange Pi Zero 3 1.5GB](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-Zero-3.html) running Armbian and it works great. Currently using a Movo LV1-USB mic, but also have had it feeding off 3 RTSP security camera streams. Not sure about power usage, but mine is powered off PoE through a PoE splitter.

**Reply by CAbing42** [reply]
me too
